### PR TITLE
[7.6][docs] 7.6.0 notes, highlights, and breaking changes

### PR DESCRIPTION
**Pull request by bmorelli25**
Cherrypick to `7.x` and `master`.

### CODE REVIEW

File: docs/guide/apm-breaking-changes.asciidoc [hunk 1]
```diff
@@ -23,6 +24,18 @@ Also see <<apm-release-notes>>.
 
 // end::notable-v8-breaking-changes[]
 
+[[breaking-7.6.0]]
+=== 7.6.0 APM Breaking changes
```
Comment (bmorelli25) [inline docs/guide/apm-breaking-changes.asciidoc]: @cauemarcondes / @ogupte  -- Any APM app breaking changes for 7.6 that should be mentioned here?

File: docs/guide/apm-release-notes.asciidoc [hunk 2]
```diff
@@ -26,6 +27,80 @@ For a full list of changes, see the
...
+You can now secure the communication between APM Agents and APM Server with
+{apm-server-ref-v}/api-key.html[API keys].
+API keys are easy to create and manage, and provide finer authorization rules than secret tokens. 
+
+SIEM integration::
+
+HTTP data collected with Elastic APM is now available in the SIEM app,
+enabling you to hunt for security threats with your APM data.
++
+See {siem-guide}/siem-ui-overview.html[SIEM UI] for information on using the SIEM app.
```
Comment (bmorelli25) [inline docs/guide/apm-release-notes.asciidoc]: @benskelker How's this look? Any recommendations?

Comment (benskelker) [reply docs/guide/apm-release-notes.asciidoc]: @bmorelli25 
Hey, sounds good.
Good that you pinged me, I should add a sentence about APM being a default data source in the SIEM app - thanks.

Comment (bmorelli25) [reply docs/guide/apm-release-notes.asciidoc]: Awesome!

File: docs/guide/apm-release-notes.asciidoc [hunk 3]
```diff
@@ -26,6 +27,80 @@ For a full list of changes, see the
...
+
+// tag::notable-v76-highlights[]
+Jaeger integration::
+
+Elastic APM now integrates with https://www.jaegertracing.io/[Jaeger], an open-source, distributed tracing system.
+This integration allows users with an existing Jaeger setup to switch from the default Jaeger backend,
+to ingesting data with Elastic's APM Server, storing data in {es}, and visualizing traces in the APM app.
+Best of all, this can be done without any instrumentation changes in your application code.
++
+See {apm-server-ref-v}/jaeger.html[Jaeger integration] for more information.
```
[the remark this answers is not in the document]
Comment (bmorelli25) [reply docs/guide/apm-release-notes.asciidoc]: Always a good to mention it. Thanks!

**Pull request merged**